# Latitude, longitude and heading feedback as reported by vehicle

std_msgs/Header header

int8 latitude_degrees   # Measured in deg
uint8 latitude_minutes  # Measured in min
uint8 latitude_seconds  # Measured in sec
int8 longitude_degrees  # Measured in deg
uint8 longitude_minutes # Measured in min
uint8 longitude_seconds # Measured in sec
float64 heading         # Measured in deg
